app/client/__tests__/dates.ts

```diff
@@ -53,11 +56,10 @@ describe("dateString", () => {
 });
 
 describe("dateIsBefore", () => {
-  const inputDate = new Date();
-  const comparisonDate = new Date();
+  const inputDate = judgementDay;
+  const comparisonDate = judgementDay;
   it("returns true if input date is before comparison date", () => {
-    comparisonDate.setMinutes(comparisonDate.getMinutes() + 5);
-    expect(dateIsBefore(inputDate, comparisonDate)).toBeTruthy();
+    expect(dateIsBefore(inputDate, new Date())).toBeTruthy();
```

Comment:
I think you may have forgotten to follow the same pattern as below on this one where you use `comparisonDate` instead of `new Date()`.

Reply:
> I think you may have forgotten to follow the same pattern as below on this one where you use `comparisonDate` instead of `new Date()`.

This one should be a valid test, we are not expecting the input and comparison date to be the same and so it will not be affected by the same issue as declaring 2 new dates side by side.   

The first test in this describe block checks that the input date (1997) is before new Date (today). The comparison date variable is used for the subsequent 2 falsey tests where we want the comparison date to less than and then equal to the input date so that we can be sure the test fails correctly.